packages/app-extensions/src/field/mappings.js

```diff
@@ -89,15 +88,19 @@ const readOnlyMapping = {
   'pulldown-date': formattedValueFactory('date'),
   'remote': formattedValueFactory('remote'),
   'search-filter': formattedValueFactory('search-filter'),
+  'serial': formattedValueFactory('string'),
   'single-remote-field': formattedValueFactory('remote'),
   'single-select-box': formattedValueFactory('single-select'),
   'sorting': formattedValueFactory('number'),
   'string': formattedValueFactory('string'),
   'text': formattedValueFactory('text'),
   'text-area': formattedValueFactory('text'),
+  'time': formattedValueFactory('time'),
+  'updatets': formattedValueFactory('datetime'),
   'upload': formattedValueFactory('document'),
   'url': formattedValueFactory('url'),
-  'uuid': formattedValueFactory('string')
+  'uuid': formattedValueFactory('string'),
+  'version': formattedValueFactory('string')
```

Comment:
Did you add these changes on purpose? They're fine, just wondering

Reply:
not really related to this change but yes, on purpose. i did some testing with the User_Test form and noticed that these types are not yet mapped.